dev/docker/ballista-builder.Dockerfile

```diff
@@ -15,18 +15,34 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM ubuntu
+FROM rust:1.63.0-buster
+
+ARG EXT_UID
+
+ENV RUST_LOG=info
+ENV RUST_BACKTRACE=full
+ENV DEBIAN_FRONTEND=noninteractive
 
 RUN apt-get update && \
-    apt-get install -y git build-essential
+    apt-get -y install libssl-dev openssl zlib1g zlib1g-dev libpq-dev cmake protobuf-compiler netcat curl unzip \
+    nodejs npm && \
```

Comment:
We probably want a newer node at some point, but I ran into a lot of trouble with every non-apt option.